examples/config.hocon.kinesis.minimal

```diff
@@ -0,0 +1,25 @@
+collector {
+  interface = "0.0.0.0"
+  port = 80
+
+  streams {
+    good = "{{good}}"
+    bad = "{{bad}}"
+
+    sink {
+      enabled = kinesis
```

Comment:
I would slightly prefer `sink.enabled` to have a default.  I think you can achieve this by having a `application.conf` file in the kinesis module (and pubsub module, etc) with kinesis-specific defaults, and a `reference.conf` file in the `core` module with all other defaults.  But I never tested if this works.